pages/planetusermanual.md

```diff
@@ -32,3 +32,72 @@ Under the PLANET system, there are two types of structures: **Nations** and **Co
   * A **Community** is not able to connect to other **Communities**.
 
 The **nations** may store thousands of Resources in many formats and languages.  Unique collections of Resources can be selected and downloaded into each **Community** from the **Nation**.  This gives a Learner a personalized approach to his or her education.
+
+## How do I navigate and use PLANET?
+###_How do I access PLANET and become a member?_
```

Comment:
Add a space between the `#`'s and the `_`  so that the headers show up right in the preview. This applies to all headers that don't have a space after `#` actually.